static/js/around-map.js

```diff
@@ -173,8 +173,22 @@ function scrollCard (poi_id) {
   window.scrollTo(0, (cardTop - cardHeight/2))  
 }
 
-// saving/adding a spot - form
+// form
+
+// display form with open/close button
+const form = document.querySelector('#poi-form')
+const displayForm = document.querySelectorAll('.display-form')
+displayForm.forEach(button => {
```

Comment:
yes I have two buttons to display with the same fonction so with `forEach` I give to both the `AddEventListener`